geoserver/webapp/pom.xml

```diff
@@ -1050,12 +1074,5 @@
         </plugins>
       </build>
     </profile>
-    <profile>
-      <id>geofence</id>
```

Comment:
why did you remove the `geofence` profile ? Even if we actually don't need some of the variables here, I think we can override the `dockerGeoserverDatadirScmVersion` here.

Also as we have discussed, why not renaming the default dockerImageName in the geofence profile by

In 15.12:
```
<dockerImageName>georchestra/geoserver:${geofence}-${project.parent.version}
```

In master:

```
<dockerImageName>georchestra/geoserver:${geofence}-latest
```